crop-blacklist.list

```diff
@@ -5,4 +5,5 @@ Vegetables
 Fibre crops
 Fibre Crops
 Flax fibre and tow
-, nes
\ No newline at end of file
+, nes
+ nes
```

Comment:
I do believe that entries with only " nes" in contrast to ", nes" are entries, where only the "nes data", i.e. the total data is available. ", nes", however, is a sum of other values… Or may I be wrong?
At least this was the reason I did not include it initially, I think.